application/view/omeka/site-admin/index/edit.phtml

```diff
@@ -53,6 +54,83 @@ if ($pages):
+                    <span class="restore-value"><?php echo $this->translate('User to be removed'); ?></span>
+                    <ul class="actions">
+                        <li><a href="#" class="o-icon-delete" aria-label="<?php echo $this->translate('Delete'); ?>"></a></li>
+                        <li><a href="#" class="o-icon-undo" aria-label="<?php echo $this->translate('Restore'); ?>"></a></li>
+                    </ul>
+                    <?php echo $user->email(); ?>
+                    <input type="hidden" name="o:site_permission[<?php echo $index; ?>][o:user][o:id]" value="<?php echo $user->id(); ?>">
+                </td>
+                <td>
+                    <input type="checkbox" name="o:site_permission[<?php echo $index; ?>][o:admin]" <?php echo $admin; ?> value="1">
+                </td>
+                <td><input type="checkbox" name="o:site_permission[<?php echo $index; ?>][o:edit]" <?php echo $edit; ?> value="1"></td>
+                <td><input type="checkbox" name="o:site_permission[<?php echo $index; ?>][o:attach]" <?php echo $attach; ?> value="1"></td>
+                <?php $index++; ?>
+            </tr>
+            <?php endforeach; ?>
+        </tbody>
+    </table>
+
+    <div class="no-resources">
+        <p><?php echo $this->translate('This site has no users. Add users using the interface to the right.'); ?></p>
+    </div>
+
+    <div id="item-set-selector" class="selector sidebar">
+        <p><?php echo $this->translate('Click on a user to add them to the site.'); ?></p>
+        <input type="text" class="selector-filter" placeholder="Filter users">
+        <ul id="new-site-user">
+            <li class="show"><?php echo $this->translate('All Users'); ?> (<?php echo count($users); ?>)
+                <ul>
+                <?php
+                    $usersByInitial = array();
+                    $initial = substr($users[0]->name(), 0, 1);
+                    foreach ($users as $user) {
+                        $userInitial = substr($user->name(), 0, 1);
+                        if ($userInitial !== $initial) {
+                            $initial = $userInitial;
+                        }
+                        $usersByInitial[$initial][] = $user;
+                    }
+                ?>
+                <?php foreach ($usersByInitial as $initial => $users): ?>
+                    <li class="selector-parent"><span>
+                        <?php echo $initial; ?></span> 
+                        (<span class="selector-child-count"><?php echo count($usersByInitial[$initial]); ?></span>)
+                        <ul>
+                        <?php foreach($users as $user): ?>
+                            <li class="selector-child" data-child-label="<?php echo $user->name(); ?>" data-user-id="<?php echo $user->id(); ?>" data-user-email="<?php echo $user->email(); ?>"><span><?php echo $user->name(); ?><br><?php echo $user->email(); ?></span></li>
```

Comment:
Ahh okay. I was wondering why the filter wasn't picking up on emails. This probably explains it.